### PR TITLE
Created service method to get classes by time slot and CRN

### PR DESCRIPTION
**Pull request by JackHarrington3**
I created a method in the service file for the course-information microservice to retrieve a list of Class objects given a time slot and CRN.

### CODE REVIEW

File: services/course-information/src/main/java/edu/uga/devdogs/course_information/service/CourseInformationService.java [hunk 1]
```diff
@@ -10,4 +11,18 @@
...
+
+    /**
+     * Method to get a list of class objects matching the given time slot and CRN. The method 
+     * it calls in the JPA layer will be implemented later.
+     * 
+     * @param timeSlot
+     * @param courseName
+     * @return A list of Class objects matching the given time slot and CRN
+     */
+    public List<Class> getClassesByCrnAndTime(String timeSlot, String crn) {
```
Comment (DogeDevYT) [inline services/course-information/src/main/java/edu/uga/devdogs/course_information/service/CourseInformationService.java:23]: Can you change this function header to 

> public List<Course> get CoursesByCrnAndTime(String timeSlot, String crn)

File: services/course-information/src/main/java/edu/uga/devdogs/course_information/service/CourseInformationService.java [hunk 2]
```diff
@@ -10,4 +11,18 @@
...
+        //Right now, the file that will contain the following method has not been created yet,
+        //so we use a fictitious file name.
+        return courseInfoJPAFile.getClassesByCrnAndTime(timeSlot, crn);
```
Comment (DogeDevYT) [inline services/course-information/src/main/java/edu/uga/devdogs/course_information/service/CourseInformationService.java:26]: Can you also change this to

> return courseInfoJPAFile.getCoursesByCrnAndTime(timeSlot, crn);

**Review by DogeDevYT — CHANGES_REQUESTED** [review]
Hey Jack, reviewed your pr and requests these changes. Just renaming the Class to Course though should be really light.

**Comment by DogeDevYT** [comment]
actually thats my bad you were right to start.

**Comment by JackHarrington3** [comment]
Thanks - I just closed the issue.

**Pull request merged**